src/app/homepage/header/versions/versions.component.ts

```diff
@@ -34,11 +34,9 @@ export class VersionsComponent implements OnInit {
     this.versionsService.addToVersionLatest().subscribe(
       (data) => {
         this.versions = data['version'];
-        console.log(data);
-        console.log(data['version']);
       },
       (error) => {
-        console.log(error);
+        throw error;
       },
```

Comment:
I put "throw" in place of console.log, so in case the NPM API does not return the latest published version of NestJS it throws an exception. If you have any ideas how to improve it let me know.

Reply:
But what happens if we throw the exception? Does it crash the component? Does it cause some unknown behavior? Would it impact the end user?

Reply:
For this part I have some ideas, I was thinking that when the api fails to hide the link which refers to the latest version of nestjs.

All the tests I've done so far, the dropdown menu works fine without any weird behavior.
To test it at the moment I'm disabling the browser safe due to cors to test it.

Reply:
@jmcdo29 I am attaching two screenshots below in case it returns an error (for example API NPM does not work) hides the link, while in the other screenshot the "latest" version of NestJS displays it (correct operation).

<img width="1624" alt="screen-ok" src="https://user-images.githubusercontent.com/11542387/160935010-13e86230-a59e-4ef5-960e-4cfbbe203d3f.png">
<img width="1826" alt="screen-error" src="https://user-images.githubusercontent.com/11542387/160935025-067922ce-15bf-434f-9ec1-74c83a570d7a.png">

If you have any ideas to improve this part further let me know. 👍

Reply:
Conflicts resolved